server/db/migrate/h2deltas/1807002_usage_data_reporting_table.sql

```diff
@@ -16,9 +16,10 @@
 
 
 CREATE TABLE UsageDataReporting (
-  id             BIGINT GENERATED BY DEFAULT AS IDENTITY ( START WITH 1 ) PRIMARY KEY,
-  serverId       VARCHAR(255) NOT NULL,
-  lastReportedAt TIMESTAMP
+  id                   BIGINT GENERATED BY DEFAULT AS IDENTITY ( START WITH 1 ) PRIMARY KEY,
+  serverId             VARCHAR(255) NOT NULL,
+  dataSharingServerUrl VARCHAR NOT NULL,
+  lastReportedAt       TIMESTAMP
 );
```

Comment:
This is not okay. Write a new delta since the original PR that introduced this table is merged already and potentially deployed at a few places.

Reply:
Also, I dont think we need to persist the url in db at all